mh/constants.py

```diff
@@ -21,10 +21,11 @@
 
 import struct
 from other.utils import pad
-from mh.time_utils import current_tick, TICKS_PER_CYCLE, get_jhen_event_times
+from mh.time_utils import current_tick, TICKS_PER_CYCLE, get_jhen_event_times,\
+    current_server_time, is_jhen_active
```

Comment:
Is importing `current_server_time` still necessary?

Reply:
Fixed